tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -105,6 +105,15 @@ public void testSingleBalFileWithConfigurables() throws BallerinaTestException {
         runLeecher.waitForText(5000);
     }
 
+    @Test
+    public void testRecordValueWithModuleClash() throws BallerinaTestException {
```

Comment:
create issue for split-package error with different variable name inside the submodule.

Reply:
created #28501